datumaro/cli/__main__.py

```diff
@@ -8,10 +8,9 @@
 import os.path as osp
 import sys
 
-from . import contexts, commands
-from .util import CliException, add_subparser
+from . import commands, contexts
 from ..version import VERSION
-
+from .util import CliException, add_subparser
```

Comment:
I'm not sure I understand the order it produces. What are the rules for `import` and `from import`?